### PR TITLE
Added GitHub/Gitlab repos

### PR DESCRIPTION
**Pull request by h3ct0r**
Hi!, in this PR I added some Git repos using the :octocat: link scheme and added a new drone company to the list. :)

edit: There is a lint warning if there is a dash in the link title and the git link is also present. 
For example this list item raise an error:
:exclamation: [VeRLab - Laboratório de Visão Computacional e Robótica (UFMG)](https://www.verlab.dcc.ufmg.br/) [:octocat:](https://github.com/verlab)

And this version does not:
:heavy_check_mark: [VeRLab](https://www.verlab.dcc.ufmg.br/) - Laboratório de Visão Computacional e Robótica (UFMG) [:octocat:](https://github.com/verlab)

Regards,

### CODE REVIEW

File: README.EN.md [hunk 1]
```diff
@@ -64,15 +65,15 @@ List of robotics initiatives in Brazil! Feel free to [contribute](CONTRIBUTING.m
 - [GRACO - Grupo de Automação e Controle (UnB)](http://www.graco.unb.br/)
 - [IDR Lab - Laboratório de Robótica Industrial e de Defesa (IME)](http://labmec.imejunior.com.br/)
 - [LABSCA - Laboratório de Sistemas Computacionais Autônomos (ITA)](http://www.comp.ita.br/labsca/)
-- [LaR - Laboratório de Robótica (UFBA)](http://www.dee.eng.ufba.br/lar/)
+- [LaR](http://www.dee.eng.ufba.br/lar/) - Laboratório de Robótica (UFBA) [:octocat:](https://github.com/lar-deeufba)
```
Comment (mateus-amarante) [inline README.EN.md:68]: I think it is better to keep it like this
```suggestion
- [LaR - Laboratório de Robótica (UFBA) ](http://www.dee.eng.ufba.br/lar/) - [:octocat:](https://github.com/lar-deeufba)
```
Placing the symbol after the dash shall not raise errors.

File: README.EN.md [hunk 2]
```diff
@@ -64,15 +65,15 @@ List of robotics initiatives in Brazil! Feel free to [contribute](CONTRIBUTING.m
...
 - [LAR - Laboratório de Robótica Aplicada Raul Guenther (UFSC)](https://robotica.ufsc.br/)
 - [LASER - Laboratório de Engenharia de Sistemas e Robótica (UFPB)](http://laser.ci.ufpb.br/)
 - [MACRO - Mechatronics, Control, and Robotics (UFMG)](http://macro.ppgee.ufmg.br/)
-  - [CORO - Laboratório de Sistemas de Computação e Robótica](http://coro.cpdee.ufmg.br/)
+  - [CORO](http://coro.cpdee.ufmg.br/) - Laboratório de Sistemas de Computação e Robótica [:octocat:](https://bitbucket.org/coroufmg)
```
Comment (mateus-amarante) [inline README.EN.md:72]:
```suggestion
  - [CORO - Laboratório de Sistemas de Computação e Robótica](http://coro.cpdee.ufmg.br/) - [:octocat:](https://bitbucket.org/coroufmg)
```

File: README.EN.md [hunk 3]
```diff
@@ -64,15 +65,15 @@ List of robotics initiatives in Brazil! Feel free to [contribute](CONTRIBUTING.m
...
 - [NAUTEC - Grupo de Automação e Robótica Inteligente (FURG)](https://nautec.furg.br/)
 - [PhiRobotics - Grupo de Pesquisa em Robótica Móvel e Visão Computacional (UFRGS)](https://www.inf.ufrgs.br/phi-group/site/)
 - [RoboFEI (FEI)](https://portal.fei.edu.br/robo-fei)
-- [VeRLab - Laboratório de Visão Computacional e Robótica (UFMG)](https://www.verlab.dcc.ufmg.br/)
+- [VeRLab](https://www.verlab.dcc.ufmg.br/) - Laboratório de Visão Computacional e Robótica (UFMG) [:octocat:](https://github.com/verlab)
```
Comment (mateus-amarante) [inline README.EN.md:76]:
```suggestion
- [VeRLab - Laboratório de Visão Computacional e Robótica (UFMG)](https://www.verlab.dcc.ufmg.br/) - [:octocat:](https://github.com/verlab)
```

File: README.md [hunk 4]
```diff
@@ -66,15 +67,15 @@ Lista de iniciativas de robótica do Brasil! Sinta-se livre para [contribuir](CO
 - [GRACO - Grupo de Automação e Controle (UnB)](http://www.graco.unb.br/)
 - [IDR Lab - Laboratório de Robótica Industrial e de Defesa (IME)](http://labmec.imejunior.com.br/)
 - [LABSCA - Laboratório de Sistemas Computacionais Autônomos (ITA)](http://www.comp.ita.br/labsca/)
-- [LaR - Laboratório de Robótica (UFBA)](http://www.dee.eng.ufba.br/lar/)
+- [LaR](http://www.dee.eng.ufba.br/lar/) - Laboratório de Robótica (UFBA) [:octocat:](https://github.com/lar-deeufba)
```
Comment (mateus-amarante) [inline README.md:70]:
```suggestion
- [LaR - Laboratório de Robótica (UFBA)](http://www.dee.eng.ufba.br/lar/) - [:octocat:](https://github.com/lar-deeufba)
```

File: README.md [hunk 5]
```diff
@@ -66,15 +67,15 @@ Lista de iniciativas de robótica do Brasil! Sinta-se livre para [contribuir](CO
...
 - [LAR - Laboratório de Robótica Aplicada Raul Guenther (UFSC)](https://robotica.ufsc.br/)
 - [LASER - Laboratório de Engenharia de Sistemas e Robótica (UFPB)](http://laser.ci.ufpb.br/)
 - [MACRO - Mechatronics, Control, and Robotics (UFMG)](http://macro.ppgee.ufmg.br/)
-  - [CORO - Laboratório de Sistemas de Computação e Robótica](http://coro.cpdee.ufmg.br/)
+  - [CORO](http://coro.cpdee.ufmg.br/) - Laboratório de Sistemas de Computação e Robótica [:octocat:](https://bitbucket.org/coroufmg)
```
Comment (mateus-amarante) [inline README.md:74]:
```suggestion
  - [CORO - Laboratório de Sistemas de Computação e Robótica](http://coro.cpdee.ufmg.br/) - [:octocat:](https://bitbucket.org/coroufmg)
```

File: README.md [hunk 6]
```diff
@@ -66,15 +67,15 @@ Lista de iniciativas de robótica do Brasil! Sinta-se livre para [contribuir](CO
...
 - [NAUTEC - Grupo de Automação e Robótica Inteligente (FURG)](https://nautec.furg.br/)
 - [PhiRobotics - Grupo de Pesquisa em Robótica Móvel e Visão Computacional (UFRGS)](https://www.inf.ufrgs.br/phi-group/site/)
 - [RoboFEI (FEI)](https://portal.fei.edu.br/robo-fei)
-- [VeRLab - Laboratório de Visão Computacional e Robótica (UFMG)](https://www.verlab.dcc.ufmg.br/)
+- [VeRLab](https://www.verlab.dcc.ufmg.br/) - Laboratório de Visão Computacional e Robótica (UFMG) [:octocat:](https://github.com/verlab)
```
Comment (mateus-amarante) [inline README.md:78]:
```suggestion
- [VeRLab - Laboratório de Visão Computacional e Robótica (UFMG)](https://www.verlab.dcc.ufmg.br/) - [:octocat:](https://github.com/verlab)
```

**Review by mateus-amarante — COMMENTED** [review]
Awesome upgrade!

Regarding the problem you mentioned, I think it is better to keep the entire name inside the link and place the symbol after the dash. In this way, they will be similar to the other items. What do you think?

I have some more repositories to add, but I think they will fit better in another PR (I'll request your review). Anyways, I think we can close #11 with this update.

**Comment by h3ct0r** [comment]
> Regarding the problem you mentioned, I think it is better to keep the entire name inside the link and place the symbol after the dash. In this way, they will be similar to the other items. What do you think?

Yes, I liked that way too! I have just made a small change to comply with the linter. If there are no more changes to do, we could merge this with main!

**Comment by mateus-amarante** [comment]
Just a comment: it is possible to group commit suggestions in batches as [this tutorial](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/incorporating-feedback-in-your-pull-request) explains

**Pull request merged**